packages/babel-traverse/src/path/context.ts

```diff
@@ -133,8 +133,8 @@ export function stop(this: NodePath) {
   this._traverseFlags |= SHOULD_SKIP | SHOULD_STOP;
 }
 
-export function setScope(this: NodePath) {
-  if (this.opts?.noScope) return;
+export function setScope(this: NodePath, ignoreNoScope?: boolean) {
```

Comment:
```suggestion
export function setScope(this: NodePath, forceScope?: boolean) {
```

`setScope` is (unfortunately) part of the public API. What do you think abuot adding an internal
```js
function _forceSetScope(path: NodePath) {}
```
instead?

Reply:
That's all good to me. :)